janitor/io.py

```diff
@@ -110,3 +111,103 @@ def read_commandline(cmd: str, **kwargs) -> pd.DataFrame:
     else:
         outcome = outcome.stdout
     return pd.read_csv(StringIO(outcome), **kwargs)
+
+
+def xlsx_table(
+    path: str,
+    sheetname: str,
+    table: Union[str, list, tuple] = None,
+) -> pd.DataFrame:
+    """
+    Returns a DataFrame of values in a table in the Excel file.
+    This applies to an Excel file, where the data range is explicitly
+    specified as a Microsoft Excel table.
+
+    If the `table` argument is provided, a pandas DataFrame is returned;
+    if the `table` argument is None, or a list/tuple of names,
+    a dictionary of DataFrames is returned, where the keys of the dictionary
+    are the table names.
+
+    Example:
+
+    ```python
+
+        filename = "excel_table.xlsx"
+
+        # single table
+        jn.xlsx_table(filename, table = 'dCategory')
+
+            CategoryID      Category
+        0           1       Beginner
+        1           2       Advanced
+        2           3      Freestyle
+        3           4    Competition
+        4           5  Long Distance
+
+        # multiple tables:
+        jn.xlsx_table(filename, table = ['dCategory', 'dSupplier'])
+
+        {'dCategory':    CategoryID       Category
+                      0           1       Beginner
+                      1           2       Advanced
+                      2           3      Freestyle
+                      3           4    Competition
+                      4           5  Long Distance,
+        'dSupplier':   SupplierID             Supplier        City State                         E-mail
+                     0         GB       Gel Boomerangs     Oakland    CA          gel@gel-boomerang.com
+                     1         CO  Colorado Boomerangs    Gunnison    CO  Pollock@coloradoboomerang.com
+                     2         CC        Channel Craft    Richland    WA                    Dino@CC.com
+                     3         DB        Darnell Booms  Burlington    VT            Darnell@Darnell.com}
+    ```
+
+    :param path: path to the Excel File.
```

Comment:
Let's standardize capitalization in these parameter descriptions? Similarly for the description of the `table` parameter.

```suggestion
    :param path: Path to the Excel File.
```